src/cavendish_particle_tracks/_main_widget.py

```diff
@@ -101,9 +101,10 @@ def __init__(self, napari_viewer: napari.viewer.Viewer):
         self.layout().addWidget(save_data_button)
 
         # Disable native napari layer controls - show again on closing this widget (hide).
-        # NB: This will break in napari 0.6.0
+        # NB: Both of these will break in napari 0.6.0
         self.viewer.window._qt_viewer.layerButtons.hide()
-
+        # Disable viewer buttons, prevents accidental crash due to viewing image stack side on.
+        self.viewer.window._qt_viewer.viewerButtons.hide()
```

Comment:
Can you put it back how you found it down on l131? The law of good manners.

Reply:
[ede0c59](https://github.com/palvarezc/cavendish-particle-tracks/pull/143/commits/ede0c59d011b12ba77fcf1514472d0bea74505ff)